src/app-root.jsx

```diff
@@ -78,6 +78,11 @@ export default class AppRoot extends React.Component {
       };
     }
 
+    jlpt5.forEach(function(entry) {entry.push("N5")});
+    jlpt4.forEach(function(entry) {entry.push("N4")});
+    jlpt3.forEach(function(entry) {entry.push("N3")});
+    jlpt2.forEach(function(entry) {entry.push("N2")});
```

Comment:
Could you change this slightly, like this:

`jlpt5.forEach((entry) => entry.push('N5'));`

Thanks!